plugins/inputs/influxdb_listener/README.md

```diff
@@ -30,14 +30,10 @@ submits data to InfluxDB determines the destination database.
   ## maximum duration before timing out write of the response
   write_timeout = "10s"
 
-  ## Maximum allowed http request body size in bytes.
-  ## 0 means to use the default of 536,870,912 bytes (500 mebibytes)
+  ## Maximum allowed HTTP request body size in bytes.
+  ## 0 means to use the default of 32MiB.
   max_body_size = 0
 
-  ## Maximum line size allowed to be sent in bytes.
-  ## 0 means to use the default of 65536 bytes (64 kibibytes)
-  max_line_size = 0
```

Comment:
Even though we are removing this option, let's keep it in the configuration so that it won't cause an error for those who already have it set.  I wrote up how I've been handling this: https://github.com/influxdata/telegraf/wiki/Deprecation#deprecate-options

Reply:
done